Box.V2/Managers/BoxFilesManager.cs

```diff
@@ -381,19 +381,20 @@ private string HexStringFromBytes(byte[] bytes)
         }
 
         /// <summary>
-        /// Retrieves a thumbnail, or smaller image representation, of this file. Sizes of 32x32, 64x64, 128x128, and 256x256 can be returned. 
-        /// Currently thumbnails are only available in .png format and will only be generated for
-        /// <see cref="http://en.wikipedia.org/wiki/Image_file_formats"/>
+        /// Retrieves a thumbnail, or smaller image representation, of this file. Sizes of 32x32,
+        ///64x64, 128x128, and 256x256 can be returned in the .png format and sizes of 32x32, 94x94, 160x160, and 320x320 can be returned in the .jpg format.
```

Comment:
Comment is copied from https://docs.box.com/reference#get-a-thumbnail-for-a-file